packages/flutter/test/material/navigation_rail_test.dart

```diff
@@ -1567,7 +1568,7 @@ void main() {
 
     await tester.pump();
     await tester.pump(const Duration(milliseconds: 100));
-    expect(rail.size.width, equals(168.0));
+    expect(rail.size.width, closeTo(216.5, 1.0));
```

Comment:
You are right, thanks for pointing that out. It was proposed in the original issue that the default animation curve should be changed to match the [Material 3 guidelines](https://m3.material.io/styles/motion/easing-and-duration/applying-easing-and-duration). Since it "only" concerns the animation, I didn't consider it a breaking change, although it breaks with existing tests. As this change implements the Material 3 guidelines, I would go forward with it. What do you think?